### PR TITLE
Collapse source and systems filter into button

### PR DESCRIPTION
**Pull request by carbonrobot**
Collapse source and system filters into a dropdown button and follow new styleguide

### Before

image

### After

image

### CODE REVIEW

File: packages/webui/src/components/ui/SourceAndSystemFilter.tsx [hunk 1]
```diff
@@ -38,129 +38,93 @@ function SourceAndSystemFilter({ className, ...props }: SourceAndSystemFilterPro
...
+          </div>
+          {hasSources ? (
+            <ul data-test-id="source-items" className="flex flex-col gap-1">
+              {connections.map(([name, isActive]) => {
+                const isSelected = filters.sources.includes(name);
+                const statusColor = isActive ? 'bg-green-400' : 'bg-red-300';
+                return (
+                  <li
+                    key={name}
+                    data-test-id="source-item"
```
Comment (carloskelly13) [inline packages/webui/src/components/ui/SourceAndSystemFilter.tsx:72]: do we want these to be enumerated per?

File: packages/webui/src/components/ui/SourceAndSystemFilter.tsx [hunk 2]
```diff
@@ -38,129 +38,93 @@ function SourceAndSystemFilter({ className, ...props }: SourceAndSystemFilterPro
...
-                </div>
-              )}
+              <ul data-test-id="system-items" className="flex flex-col gap-1">
+                {systems.map(system => {
+                  const isSelected = filters.systems.includes(system.name);
+                  const icon = system.getIconUri?.() ?? defaultIcon;
+                  return (
+                    <li
+                      key={system.name}
+                      data-test-id="system-item"
```
Comment (carloskelly13) [inline packages/webui/src/components/ui/SourceAndSystemFilter.tsx:108]: Same here

**Review by carloskelly13 — APPROVED** [review]
🎉

File: packages/webui/src/components/ui/SourceAndSystemFilter.tsx (hunk 1 above)
Comment (kgpax) [reply packages/webui/src/components/ui/SourceAndSystemFilter.tsx:72]: It's normally enough to have these share the test id, and then we can both count the number of items, and iterate using `.at(index)`.

Comment (carloskelly13) [reply packages/webui/src/components/ui/SourceAndSystemFilter.tsx:72]: Oh okay, that makes sense too!

**Pull request merged**